microsoft_atp/plugin.spec.yaml

```
@@ -594,6 +722,100 @@ actions:
         description: List of machines with provided software
         type: '[]machine_software'
         required: true
+  blacklist:
+    title: Blacklist
+    description: Submit or update new indicator
+    input:
+      indicator:
+        title: Indicator
+        description: The value of the indicator
+        type: string
+        example: 220e7d15b011d7fac48f2bd61114db1022197f7f
+        required: true
+      indicator_type:
+        title: Indicator Type
+        description: Type of the indicator
+        type: string
+        example: Domain
+        required: false
+        enum:
+          - IpAddress
+          - Url
+          - Domain
+          - FileSha1
+          - FileSha256
+      action:
+        title: Action
+        description: The action that will be taken if the indicator will be discovered in the organization
+        type: string
+        example: AlertAndBlock
+        default: AlertAndBlock
+        required: false
+        enum:
+          - Alert
+          - AlertAndBlock
+          - Allowed
+      application:
+        title: Application
+        description: The application associated with the indicator
+        type: string
+        example: demo-test
+        required: false
+      title:
+        title: Title
+        description: Indicator alert title
+        type: string
+        example: test
+        required: false
+      description:
+        title: Description
+        description: Description of the indicator
+        type: string
+        example: description
+        required: false
+      expiration_time:
+        title: Expiration Time
+        description: The expiration time of the indicator
+        type: string
+        example: "2020-12-12T00:00:00Z"
+        required: false
+      severity:
+        title: Severity
+        description: The severity of the indicator
+        type: string
+        example: High
+        default: High
+        required: false
+        enum:
+          - Informational
+          - Low
+          - Medium
+          - High
+      recommended_actions:
+        title: Recommended Actions
+        description: TI indicator alert recommended actions
+        type: string
+        example: nothing
+        required: false
+      rbac_group_names:
+        title: RBAC Group Names
+        description: List of RBAC group names the indicator would be applied to
+        type: '[]string'
+        example: [group1,group2]
```

Needs to be a valid JSON array
```suggestion
        example: '["group1","group2"]'
```